editor/editor_node.cpp

```diff
@@ -103,6 +102,8 @@
 #include "editor/import/resource_importer_texture.h"
 #include "editor/import/resource_importer_texture_atlas.h"
 #include "editor/import/resource_importer_wav.h"
+#include "editor/import/scene_importer_mesh.h"
```

Comment:
This doesn't seem used directly, the `_node_3d.h` include is sufficient no?

Same in other files which include both.

Reply:
Removed.